_docs/releasing_version.md

```diff
@@ -29,13 +57,15 @@ ruby support has been dropped.)
 If necessary, run `bundle install` before continuing. If you want you can also only install the needed gems:
 
 ```bash
-bundle install --path .vendor/ --without system_tests development
+bundle config set --local path 'vendor'
+bundle install --without system_tests development
 ```
 
 And in case you installed the gems before:
 
 ```bash
-bundle install --path .vendor/ --without system_tests development; bundle update; bundle clean
+bundle config set --local path 'vendor'
+bundle install --without system_tests development; bundle update; bundle clean
```

Comment:
if you switch to bundle config, also do it like this:
`bundle config set --local without 'development system_tests'`